src/text/text_data_source.rs

```diff
@@ -1,42 +0,0 @@
-use std::io;
-use std::io::{BufRead, BufReader};
-
-/// Types that implement this trait can be converted into an implementation of [io::BufRead],
-/// allowing them to be passed to the [TextReader::new] constructor. This allows [TextReader::new]
-/// to support reading Strings, &str slices, &[u8] slices, files, etc.
-pub trait TextIonDataSource {
```

Comment:
🗺️ The renamed version of this trait/file appeared earlier in the diff: `ToIonDataSource`.